docs/docs/en/guides/quick-start/install-tuist.md

```diff
@@ -41,12 +41,24 @@ brew install --formula tuist
 brew install --formula tuist@x.y.z
 ```
 
+::: tip VERIFYING THE AUTHENTICITY OF THE BINARIES
+You can verify that your installation's binaries have been built by us by running the following commands and checking that the team ID is `U6LC622NKF`:
```

Comment:
What do you think about updating the bash commands, so that they fail if the team ID's is not the one we expect?